Tutorials/intro-to-QUAlibrate/pyproject.toml

```diff
@@ -0,0 +1,47 @@
+[project]
+name = "superconducting-calibrations"
+version = "0.1.0"
+description = "QM Superconducting Calibration Graphs"
+readme = "README.md"
+license = { text = "BSD-3-Clause" }
+requires-python = ">=3.9,<3.13"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+]
+dependencies = [
```

Comment:
Need to update the dependencies (remove quam, qiskit, quam-builder, xarray...)

Reply:
updated to the minimal needed things (quam-building was needed to the `CommonNodeParameters`)